lib/userInfo.ts

```diff
@@ -5,6 +5,14 @@ export const getUserInfo = async (address: string) => {
   return data
 }
 
+export const getSimilarProfiles = async (address: string) => {
+  const { data } = await axios.get(
+    `/api/v2/get/userprofile/similarProfiles?walletAddress=${address}`,
```

Comment:
for one query param this is fine. We really should be leveraging the params object in axios. We get lazy and do not do this, but it feels  cleaner to me.